File: src/main/java/net/greemdev/core/listeners/MulticraftListener.java
package net.greemdev.core.listeners;

import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.server.ServerCommandEvent;

public class MulticraftListener implements Listener {

    //Stop Multicraft's annoying ass /list spam; because apparently there's no easier way for Multicraft to check if the server is online.
    @EventHandler
    public void onServerCommand(ServerCommandEvent event) {
        if (event.getCommand().equalsIgnoreCase("list")) {
            event.setCancelled(true);
            System.out.print(" ");
        }
    }

}

File: src/main/kotlin/net/greemdev/core/listeners/MulticraftListener.kt
package net.greemdev.core.listeners

import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.server.ServerCommandEvent

public class MulticraftListener: Listener {

    @EventHandler
    public fun onServerCommand(event: ServerCommandEvent) {
        if (event.command == "list") {
            event.isCancelled = true;
            print(" ")
        }
    }

}